repose-aggregator/tests/test-bundles/filter-one/build.gradle

```diff
@@ -1,6 +1,6 @@
 dependencies {
-    provided 'javax.servlet:javax.servlet-api'
-    provided 'javax.inject:javax.inject'
+    compileOnly 'javax.servlet:javax.servlet-api'
```

Comment:
lol, you moved all the others to the standard order but not this one and three

Reply:
D'oh!